src/pretix/presale/views/event.py

```diff
@@ -104,6 +108,29 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class EventIcalDownload(EventViewMixin, View):
+    def get(self, request, *args, **kwargs):
+        if not self.request.event:
+            raise Http404(_('Unknown event code or not authorized to access this event.'))
+
+        cal = Calendar()
+        cal.add('version', '2.0')
+        cal.add('prodid', '-//{}//{}//'.format(self.request.event.organizer.name, self.request.event.name))
```

Comment:
this is the ical spec.
[Product Identifier Spec](https://icalendar.org/iCalendar-RFC-5545/3-7-3-product-identifier.html)

>This property specifies the identifier for the product that created the iCalendar object.

I think `prodid` should contain system information instead of event information? I couldn't find any restriction on each parameter of `prodid` in the spec., so we can drop it at this moment i guess? It's just the identifier of the ical object anyway.